### ISSUE TITLE
Add transaction feature 

### ISSUE THREAD

**Issue by publicocean0** [Fullstop000/wickdb#44]
It is missing this important feature

**Comment by Fullstop000** [comment]
Transaction may not be suitable to put into a key-value engine directly. I think we can build this on top of it :).

**Comment by publicocean0** [comment]
yes it could be a additional optional  layer wrapping the basic version
